src/lib/emailTemplates.ts

```
@@ -483,20 +484,47 @@ export const createBusinessVerificationEmail = (businessDetails: {
       `;
       break;
     case 'documents_required':
-      subject = 'Documents Required for Business Verification - Rainbow Paws';
+      subject = 'Specific Documents Required for Business Verification - Rainbow Paws';
+
+      // Format required documents list
+      let documentsList = '';
+      if (businessDetails.requiredDocuments && businessDetails.requiredDocuments.length > 0) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)